### ISSUE TITLE
Investigate ACT rules that assume HTML 5 vs old rule techniques and tutorials that assume HTML 4, open issues with W3C? - 3687

### ISSUE THREAD

**Issue by aliunwala** [IBMa/equal-access#598]
investigate where W3C WCAG 2.1 guidance, rules, techniques, and tutorials lag behind HTML 5.

### Describe what needs to be accomplished
_investigate opening issues against W3C ACT rules, down-level techniques and/or tutorials._
_Tom said: Didn't we get this rule [table_headers_ref_valid] from ACT? Do we need to open an issue with them?_

* [ ] https://act-rules.github.io/rules/a25f45 "headers attribute specified on a cell refers to cells in the same table element"
* [ ] other rules?
* [ ] https://www.w3.org/WAI/tutorials/tables/multi-level/
* [ ] [H43: Using id and headers attributes to associate data cells with header cells in data tables](https://www.w3.org/WAI/WCAG21/Techniques/html/H43)
* [x] [H63: Using the scope attribute to associate header cells and data cells in data tables](https://www.w3.org/WAI/WCAG21/Techniques/html/H63)

### Why does this need to be accomplished
_Keep Checker rules in sync with W3C._

### Related issues or references
See 

1. Create unpublished ACT Rule a25f45: headers attribute specified on a cell refers to cells in the same table element #3138 https://github.ibm.com/IBMa/E2E/issues/3138

1. box note: https://ibm.ent.box.com/notes/859497998779

1. Help content for Create unpublished ACT Rule a25f45: headers attribute #3138 #3646  https://github.ibm.com/IBMa/E2E/issues/3646

1. slack thread https://ibm-studios.slack.com/archives/GSGN38DQE/p1633728054078000

Link to original issue:
https://github.ibm.com/IBMa/E2E/issues/3687

Image of original issue:
![image.png](https://images.zenhubusercontent.com/61bb8e6b2e4f8534832597ac/6b42eb92-d216-4d69-be7b-d9e01d84e44c)

**Comment by aliunwala** [comment]
Comment by Phil:
Mike opened one issue against W3C so far

Sufficient technique H63 should incorporate HTML5 and address missing info 2077
https://github.com/w3c/wcag/issues/2077

**Comment by aliunwala** [comment]
Commnet by Marc:
@pjenkins @michael-gower 

Tom also suggested - can't we just open one blanket issue with W3C/ACT asking THEM to investigate this on their end?  We can provide the 3 or 4 examples that we're already aware of - and then close this issue on our end.

**Comment by aliunwala** [comment]
From triage:
This is probably ACT's work not ours. So we should open an issue with ACT to do this for us.
